blog/2025-04-11-wpewebkit-2.48.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "WPE WebKit 2.48 highlights"
+author: csaavedra
+permalink: /blog/2025-04-11-wpewebkit-2.48.html
+preview: A review of the most significant changes in WPE WebKit 2.48
+thumbnail: /assets/img/logo-black.svg
+---
+
+The WPE WebKit team has been working hard during the past six months and has recently released version 2.48 of the WPE port of WebKit. As is now tradition, here is an overview of the most important changes in this new stable release series.
+
+## Graphics and Rendering
+
+The work on the graphics pipeline of WebKit continues, and a lot of improvements, refactorings, bug fixes, and optimizations have happened under the hood. These changes bring both performance and rendering improvements, and are too many to list individually. Nonetheless, there are a number of  interesting changes.
+
+### GPU Worker Threads
+
+When GPU rendering with Skia is in use&mdash;which is the default&mdash;, tiles will be rendered in worker threads, which has a positive impact on performance. Threads were already used when using the CPU for rendering.
```

Comment:
Currently GPU rendering is **not yet** the default for WPE (it is for the GTK port).

```suggestion
When GPU rendering with Skia is in use, tiles will be rendered in worker threads, which has a positive impact on performance. Threads were already used when using the CPU for rendering. Note that GPU usage for rendering is not yet the default in the WPE port, but may be enabled setting `WEBKIT_SKIA_ENABLE_CPU_RENDERING=0` in the environment before running programs that use WPE WebKit.
```